pkg/performanceprofile/controller/utils.go

```diff
@@ -0,0 +1,51 @@
+package controller
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	olmoperators "github.com/operator-framework/api/pkg/operators/install"
+	olmv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+)
+
+func ListPerformanceOperatorCSVs(k8sclient client.Client, options []client.ListOption, paginationLimit uint64) ([]olmv1alpha1.ClusterServiceVersion, error) {
+	// Register OLM types to the client
+	olmoperators.Install(k8sclient.Scheme())
```

Comment:
this should preferably be done once. Now, this function is still likely called once, but still.

Reply:
Totally agree ... 
My concern here is that this needs to be executed for the `List` operation to work, so either the function execute it or we assume it is a pre-requisite for the function to work. 

I do not know how to test if the scheme is properly configured so opted for this.